spec/reek/source/source_code_spec.rb

```diff
@@ -6,20 +6,20 @@
   describe '#syntax_tree' do
     it 'associates comments with the AST' do
       source = "# this is\n# a comment\ndef foo; end"
-      source_code = Reek::Source::SourceCode.new(source, '(string)')
+      source_code = Reek::Source::SourceCode.new(code: source, origin: '(string)')
```

Comment:
This **is** actually more readable than before, but like explained above you can switch to keyword arguments and then keep the exact same syntax here ;)